client/components/common/TestPlanResultsTable/index.jsx

```diff
@@ -14,9 +14,16 @@ const renderAssertionRow = (assertionResult, priorityString) => {
     );
 };
 
-const TestPlanResultsTable = ({ test, testResult, tableClassName = '' }) => {
+const TestPlanResultsTable = ({
+    test,
+    testResult,
+    tableClassName = '',
+    isReportsPage = false
+}) => {
     return (
         <>
+            {isReportsPage ? <h3>Results for each command</h3> : null}
```

Comment:
If this headline is not always needed maybe it's a sign it doesn't belong in this component and can be handled by the parent component.

Reply:
Good point, addressed in https://github.com/w3c/aria-at-app/pull/845/commits/9eb58751f7aabdd88fe8aaf53d1afe21d67d7b7f.